### PR TITLE
fix(ActivityTab): vertically center empty content

### PR DESCRIPTION
**Pull request by edward-ly**
Fixes an issue from a design review for another app that was recently brought to my attention: `-- 8 comments app: The ‘No activity’ text and logo is too close to the input line. Some space is needed`.

### CODE REVIEW

File: src/views/ActivityTab.vue [hunk 1]
```diff
@@ -4,7 +4,7 @@
 -->
 
 <template>
-	<div :class="{ 'icon-loading': loading }">
+	<div :class="{ 'activity': true, 'icon-loading': loading }">
```
Comment (artonge) [inline src/views/ActivityTab.vue:7]:
```suggestion
	<div :class="{ 'icon-loading': loading }"
		class="activity">
```

**Review by artonge — COMMENTED** [review]
Thanks ! :)
Can you also add before/after screenshots?

File: src/views/ActivityTab.vue (hunk 1 above)
Comment (edward-ly) [reply src/views/ActivityTab.vue]: Thanks, I forgot it is possible for both can be declared at once.

**Comment by edward-ly** [comment]
> Thanks ! :) Can you also add before/after screenshots?

Sorry, I forgot about that, although I am having trouble getting the comments app to behave as expected on my end.

This is a screenshot that was provided to me by @marcelklehr:


And here is the before and after from my setup.
Screenshot 2024-07-22 at 08-08-51 Files - Nextcloud
Screenshot 2024-07-22 at 08-10-06 Files - Nextcloud

**Review by artonge — APPROVED** [review]

**Comment by artonge** [comment]
Red CI is unrelated, merging.

**Pull request merged**